templates/blog/post_list.html

```diff
@@ -0,0 +1,15 @@
+<main class="container">
+    <div class="row">
+        <div class="col">
+            {% for post in posts %}
+                <article class="post">
+                    <time class="date">
+                        {{ post.published_date }}
```

Comment:
Does the tutorial solve the case of a not yet published post (a draft)? Or a post scheduled for the future?

Reply:
I didn't find this. Should I also integrate this?

Reply:
This can be postponed as a future improvement, I suppose.

Reply:
Ok

Reply:
Marking this thread as resolved.